.drone.star

```diff
@@ -2,7 +2,7 @@ def main(ctx):
   versions = [
     {
       'value': 'latest',
-      'qa': 'https://download.owncloud.org/community/testing/owncloud-10.4.0-qa.tar.bz2',
+      'qa': 'https://download.owncloud.org/community/daily/owncloud-daily-master-qa.tar.bz2',
```

Comment:
Actually we might not need this change. The 10.4.0 QA tarball should still have OK test code, which will have the correct test scenarios for 10.4.0. We just need to get the correct Behat `composer.json` to get better dependencies.